app/views/patients/show.html.erb

```diff
@@ -128,70 +128,12 @@
-                              assigned_users: @possible_assigned_users
-                            }) %>
-      </div>
-    </div>
-    <table class="cc_table table table-sm table-striped table-bordered table-hover table-smaller-font" style="width:100%">
-      <thead>
-        <tr>
-          <th class="DataTable-table-header" style="min-width: 140px;">Actions</th>
-          <th class="DataTable-table-header">First Name</th>
-          <th class="DataTable-table-header">Last Name</th>
-          <th class="DataTable-table-header">Phone Number</th>
-          <th class="DataTable-table-header">Email</th>
-          <th class="DataTable-table-header">Last Date of Exposure</th>
-          <th class="DataTable-table-header">Assigned User</th>
-          <th class="DataTable-table-header">Contact Attempts</th>
-          <th class="DataTable-table-header">Enrolled?</th>
-          <th class="DataTable-table-header">Notes</th>
-        </tr>
-      </thead>
-      <tbody>
-        <% @close_contacts.each do |cc| -%>
-        <tr>
-          <td>
-            <div class="btn-group btn-group-sm" role="group">
-              <%= react_component('patient/close_contact/CloseContact', {
-                                    authenticity_token: form_authenticity_token,
-                                    patient: @patient,
-                                    close_contact: cc,
-                                    can_enroll_patient_close_contacts: current_user.can_enroll_patient_close_contacts?,
-                                    assigned_users: @possible_assigned_users
-                                  }) %>
-            </div>
-          </td>
-          <td><%= cc.first_name %></td>
-          <td><%= cc.last_name %></td>
-          <td><%= cc.primary_telephone.present? && cc.primary_telephone.length > 10 ? cc.primary_telephone&.gsub('+1', '')&.gsub(/\D/, '')&.insert(3, '-')&.insert(7, '-') : '' %></td>
-          <td><%= cc.email %></td>
-          <td><%= cc.last_date_of_exposure&.to_date&.strftime('%m/%d/%Y') %></td>
-          <td><%= cc.assigned_user %></td>
-          <td><%= cc.contact_attempts || 0 %></td>
-          <td><%= cc.enrolled_id ? 'Yes' : 'No' %> </td>
-          <td class="wrap-words"><%= cc.notes && cc.notes.length > 500 + 3 ? (cc.notes[0..500] + '...') : cc.notes %></td>
-        </tr>
-        <% end -%>
-      </tbody>
-    </table>
-    <% if @close_contacts.empty? %>
-      <div class="text-center mb-4">No data available in table.</div>
-    <% end %>
-  </div>
-</div>
+  <%= react_component('patient/close_contacts/CloseContactTable', {
+                        patient: @patient,
+                        can_enroll_close_contacts: current_user.can_enroll_patient_close_contacts?,
+                        authenticity_token: form_authenticity_token,
+                        assigned_users: @patient.jurisdiction.assigned_users
```

Comment:
we already make this query in the controller

```suggestion
                        assigned_users: @possible_assigned_users
```

Reply:
fixed